src/Logging.jl

```diff
@@ -123,7 +123,7 @@ write_log(syslog::SysLog, color::Symbol, msg::AbstractString) = send(syslog.sock
 write_log{T<:IO}(output::T, color::Symbol, msg::AbstractString) = (println(output, msg); flush(output))
 write_log(output::Base.TTY, color::Symbol, msg::AbstractString) = (Base.println_with_color(color, output, msg); flush(output))
 
-function formatlog(format::AbstractString, logging_fields::Dict)
+function formatlog(format::AbstractString, logging_fields::Dict{AbstractString,ANY})
```

Comment:
Not sure about the `ANY` here, but my understanding is that this will let `logging_fields` be a Dict with a key of Abstract String, and any type of value (but not necessarily the Any type).

I figured since this function itself doesn't depend on any type in particular this would be the most "open", but I could be totally off.